README.rst

```diff
@@ -1,3 +1,7 @@
+**WARNING**: This Structural Imbalance demo has been relocated to and will be maintained in the
+`dwavesystems/demos <https://github.com/dwavesystems/demos/tree/master/structural-imbalance>`_
+repo. The existing repository seen on this page will no longer be updated.
+
```

Comment:
How about: "**Attention**: This repository is obsolete. The Structural Imbalance demo is now maintained here: `dwavesystems/demos <https://github.com/dwavesystems/demos/tree/master/structural-imbalance>`_\ ."   ?